### PR TITLE
Fixed login issue

### PR DESCRIPTION
**Pull request by agonxalez**
:cries:

### CODE REVIEW

**Comment by blamboy090** [comment]
If I am logged out on one browser and copy paste a profile page link into the browser it takes you to the below screen shot:
using url

**Comment by lsanchezcode** [comment]
Same issue, that is the only broken one for me!

File: pages/user/[id].tsx [hunk 1]
```diff
@@ -16,22 +17,26 @@ const Home: NextPage = () => {
...
+    let calledPushLatest;
+    setCalledPush((latest) => {
+      calledPushLatest = latest;
+      return latest;
+    });
+    if (calledPushLatest) return;
+
+    if (!isLoading && !user) {
+      router.push('/');
+    } else if (!isLoading) {
```
Comment (StephenBecker-catalyte) [inline pages/user/[id].tsx:33]: can take out the console log and the is loading check, it is caught by line 21

    if (!user) {
      router.push('/');
    }

**Review by StephenBecker-catalyte — APPROVED** [review]
minor suggestions otherwise LGTM

**Comment by blamboy090** [comment]
LGTM! Excellent work!

**Comment by lsanchezcode** [comment]
LGTM! :)

**Review by blamboy090 — APPROVED** [review]
LGTM!

**Pull request merged**